xl2times/utils.py

```diff
@@ -10,6 +11,10 @@
 
 from . import datatypes
 
+# prevent excessive number of processes in Windows and high cpu-count machines
+# TODO make this a cli param or global setting?
+max_workers: int = 4 if os.name == "nt" else min(16, os.cpu_count() or 16)
```

Comment:
I have 32-cores, and without this the full benchmark suite was spawning 32*32 processes when run.
Probably a bigger deal on windows, but I figured it'd be worth capping on other OSes too. 
Speeds are about the same with/without on linux.